frontend/src/libhoclient/host_orchestrator_client.go

```diff
@@ -386,6 +398,55 @@ func (c *HostOrchestratorServiceImpl) ListCVDs() ([]*hoapi.CVD, error) {
 	return res.CVDs, nil
 }
 
+func (c *HostOrchestratorServiceImpl) Powerwash(group, name string) error {
```

Comment:
nit: Rename `group, name` as `group, instance` or `groupName, instanceName`.

Reply:
this code was removed from this PR following the comment below. I'll address the comment in the following PR containing this change.